src/VirtualClient/TestResources/Results_StressAppTest.txt

```diff
@@ -0,0 +1,26 @@
+2022/12/28-08:15:00(UTC) Log: Commandline - ./stressapptest -s 60 -l logfile_v8_pass.txt
+2022/12/28-08:15:00(UTC) Stats: SAT revision 1.0.9_autoconf, 64 bit binary
+2022/12/28-08:15:00(UTC) Log: adityaa @ adityaa-vm-lin-ubu20-d4 on Fri Oct 21 08:05:58 UTC 2022 from open source release
+2022/12/28-08:15:00(UTC) Log: 1 nodes, 16 cpus.
+2022/12/28-08:15:00(UTC) Log: Defaulting to 16 copy threads
+2022/12/28-08:15:00(UTC) Log: Total 64318 MB. Free 62116 MB. Hugepages 0 MB. Targeting 60910 MB (94%)
+2022/12/28-08:15:00(UTC) Log: Prefer plain malloc memory allocation.
+2022/12/28-08:15:00(UTC) Log: Using mmap() allocation at 0x7fe9f022e000.
+2022/12/28-08:15:00(UTC) Stats: Starting SAT, 60910M, 60 seconds
+2022/12/28-08:15:03(UTC) Log: Region mask: 0x1
+2022/12/28-08:15:13(UTC) Log: Seconds remaining: 50
+2022/12/28-08:15:23(UTC) Log: Seconds remaining: 40
+2022/12/28-08:15:33(UTC) Log: Seconds remaining: 30
+2022/12/28-08:15:43(UTC) Log: Seconds remaining: 20
+2022/12/28-08:15:53(UTC) Log: Seconds remaining: 10
+2022/12/28-08:16:04(UTC) Stats: Found 0 hardware incidents
+2022/12/28-08:16:04(UTC) Stats: Completed: 4582288.00M in 60.00s 76365.33MB/s, with 0 hardware incidents, 0 errors
+2022/12/28-08:16:04(UTC) Stats: Memory Copy: 4582288.00M at 76369.52MB/s
+2022/12/28-08:16:04(UTC) Stats: File Copy: 0.00M at 0.00MB/s
```

Comment:
What work would be needed to enable file, networking etc?

Reply:
StressAppTest supports some parameters like "ipaddr" to add a network thread connecting to system, and some parameters which can be used to configure disk related operations. Having said that, the primary use-case of StressAppTest is to stress out the memory for stability testing.
We have the flexibility to add these extra parameters in commandLine (through profile) in case some user wants to configure these.
I am resolving this comment for now. Please let me know if any modifications are needed.